org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/validation/AbstractValidatorConfigurationBlock.java

```diff
@@ -99,7 +99,6 @@ private Composite createStyleTabContent(Composite folder) {
 
 	/**
 	 * This implementation returns Dialog settings which were bind in {@link org.eclipse.xtext.ui.DefaultUiModule}
-	 * normally {@link org.eclipse.ui.plugin.AbstractUIPlugin#getDialogSettings()}<br>
```

Comment:
maybe we also can or should change severity to warning

Reply:
but i did not find an option

Reply:
just `org.eclipse.jdt.core.compiler.problem.invalidJavadocTagsDeprecatedRef=disabled`

Reply:
I also do not see an option. I have raised feature request https://bugs.eclipse.org/bugs/show_bug.cgi?id=567968 for that.
For now, I would not remove the line, but maybe just change `@link` to `@code`.

Reply:
I was too eager. The option `org.eclipse.jdt.core.compiler.problem.invalidJavadocTagsDeprecatedRef=disabled`does it.

(Uncheck option "Report deprecated references")

Reply:
with disabled i dont get a warning

Reply:
Yes, that's the consequence. Maybe the options "Report non visible/deprecated references" should also get severity drop-downs?

Reply:
Or we go with setting all to warning. We might oversee real errors, but ATM there are none and we just have to have an eye on new warnings. Removing the comment line is IMO no option.

Reply:
used code instead of link